gcloud-java-storage/src/main/java/com/google/gcloud/storage/Storage.java

```diff
@@ -695,6 +695,10 @@ public static BlobListOption prefix(String prefix) {
 
     /**
      * Returns an option to specify whether blob listing should include subdirectories or not.
+     * {@link StorageOptions#pathDelimiter()} is used as a path delimiter. If set to {@code true}
+     * also blobs in subdirectories are listed. If set to {@code false} and used in combination
```

Comment:
Since we're not really talking about "directories" it might be good to be more precise here and avoid using "directories" and "subdirectories" without more context. For example I noticed different behavior depending on whether my prefix ended with a trailing `/` or not, which I didn't see documented anywhere.